lms/static/scripts/frontend_apps/components/ErrorDialogApp.js

```diff
@@ -0,0 +1,65 @@
+import { Fragment, createElement } from 'preact';
+import { useContext } from 'preact/hooks';
+
+import { Config } from '../config';
+
+import ErrorDisplay from './ErrorDisplay';
+import Dialog from './Dialog';
+
+/**
+ * Error dialog displayed when authorization with a third-party API via OAuth
+ * fails.
+ *
+ */
+export default function ErrorDialogApp() {
```

Comment:
My thinking here is that this could be used for any error, or rather any error that 's not oauth related (covered by `OAuth2RedirectErrorApp`)

Reply:
This seems like a reasonable generalization, although we don't know for sure that it is the right one until we have other use cases in future.